### PR TITLE
Issue 1 user can see big three

### PR DESCRIPTION
**Pull request by vkahlon**
Demo

Desktop:

https://user-images.githubusercontent.com/47346471/147996542-dce5751f-d215-44cc-9a8b-7d923dc21d5c.mp4

Mobile:

https://user-images.githubusercontent.com/47346471/147996564-3a278cae-d24c-4fd0-a176-889bc14aa3ef.mp4

### CODE REVIEW

**Comment by vkahlon** [comment]
what it looks like when change is positive

Screen Shot 2022-01-03 at 5 22 12 PM

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    btcPrice = Math.round(btcPrice * 100) / 100;
+    $changeBtcPrice.textContent = btcPrice;
+
+    var btcVol = newReq.response.ticker.volume;
+    btcVol = Math.round(btcVol);
+    $changeBtcVolume.textContent = btcVol;
+
+    var btcChange = newReq.response.ticker.change;
+    btcChange = Math.round(btcChange * 100) / 100;
+    $changeBtc.textContent = btcChange;
```
Comment (codingcodymiller) [inline js/main.js:19]: I would make this into a named function that generates a DOM tree and `return`s it for a single ticker. It should have an argument of a single object for a single ticker, which in your case would be `newReq.ticker.price`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by vkahlon** [comment]
Screen Shot 2022-01-04 at 9 46 32 AM

File: index.html [hunk 2]
```diff
@@ -19,91 +19,10 @@
...
-              </tr>
-              <tr>
-                <td class="description">Change</td>
-                <td id="link-change" ></td>
-              </tr>
-            </tbody>
-          </table>
-        </div>
-      </div>
+      <div style="justify-content: center;" class="main-table row" data-view="big-three"></div>
```
Comment (codingcodymiller) [inline index.html:22]: Nooooo, a `style` attribute?? Make a class for this.

File: js/main.js [hunk 3]
```diff
@@ -1,71 +1,158 @@
...
     var btcChange = newReq.response.ticker.change;
     btcChange = Math.round(btcChange * 100) / 100;
-    $changeBtc.textContent = btcChange;
+    btcObject.base = btcBase;
+    btcObject.price = btcPrice;
+    btcObject.volume = btcVol;
+    btcObject.change = btcChange;
+    btcObject.target = btcTarget;
+    btcObject.color = btcColor;
+    displayBigThreeTable(btcObject, btcColor);
```
Comment (codingcodymiller) [inline js/main.js:22]: What is the purpose of the second argument here? Your function definition only has one parameter.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 3 above)
Comment (vkahlon) [reply js/main.js]: d'oh

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**